### PR TITLE
What is currently enabling our ability to predict network structure?

### PR DESCRIPTION
**Pull request by FrancisBanville**
added placeholder text

### CODE REVIEW

**Review by TanyaS08 — APPROVED** [review]
I ❤️ the inclusion of all the OA databases!

File: manuscript.md [hunk 1]
```diff
@@ -14,6 +14,37 @@
...
+
+The rise of computing power, along with recent advances in machine learning techniques 
+and applications [see @Christin2019AppDee for the use of deep learning in ecology], 
+enable us to manipulate a very large number of data from different sources. Accurate 
+predictions of ecological networks across space can thus be generated by integrating 
+various high-quality, open access datasets, such as the ones archived in the abovementioned 
+databases, if we use predictive methods appropriately. Moreover, ongoing developments in 
+the field of artificial intelligence are aimed at using deep learning more efficiently in 
+low-data regimes [e.g. @Antoniou2018DatAug] and with unbalanced datasets [@Chawla2010DatMin]. 
+Considering the current biases in network ecology [@Poisot2020EnvBia] and the scarcity of data
```
Comment (TanyaS08) [inline manuscript.md:45]: we could maybe add in a bit about coupled with the growth of OA databases the future looks very positive?

File: manuscript.md [hunk 2]
```diff
@@ -14,6 +14,37 @@
 
 ## What is currently enabling our ability to predict network structure?
 
+The acquisition of biodiversity and environmental data has tremendously 
+increased over the past decades thanks to the rise of citizen science 
+[@Dickinson2010CitSci] and of novel technology [@Stephenson2020TecAdv], 
+including wireless sensors [@Porter2005WirSen], DNA monitoring 
+[@Creer2016EcoSF], and satellite remote sensing [@Skidmore2015AgrBio; @Lausch2016LinEar]. 
+Standard practices in data integration and quality control [@Kissling2018BuiEss] and in
```
Comment (TanyaS08) [inline manuscript.md:22]: We could maybe also include standardised sampling protocols e.g. plant functional traits [Pérez-Harguindeguy et. al., (2016)](https://www.publish.csiro.au/bt/pdf/BT12225) and metadata e.g. [DarwinCore](https://www.tdwg.org/) as this also helps promote interoperability making it easier to integrate multiple data sources

File: manuscript.md [hunk 3]
```diff
@@ -14,6 +14,37 @@
...
+The rise of computing power, along with recent advances in machine learning techniques 
+and applications [see @Christin2019AppDee for the use of deep learning in ecology], 
+enable us to manipulate a very large number of data from different sources. Accurate 
+predictions of ecological networks across space can thus be generated by integrating 
+various high-quality, open access datasets, such as the ones archived in the abovementioned 
+databases, if we use predictive methods appropriately. Moreover, ongoing developments in 
+the field of artificial intelligence are aimed at using deep learning more efficiently in 
+low-data regimes [e.g. @Antoniou2018DatAug] and with unbalanced datasets [@Chawla2010DatMin]. 
+Considering the current biases in network ecology [@Poisot2020EnvBia] and the scarcity of data 
+of species interactions, the prediction of ecological networks will undoubtedly benefit from these improvements.
```
Comment (TanyaS08) [inline manuscript.md:46]: we could maybe add in a bit about coupled with the growth of OA databases the future looks very positive?

Comment (gottacatchenall) [reply manuscript.md:46]: also some stuff about the [increased interest in Biodiversity Observation Networks](https://doi.org/10.1016/j.cosust.2011.12.005) at the governmental level and the [recent focus on open data](dx.doi.10.1016/j.tree.2019.04.005) could be good additional cites

**Review by gottacatchenall — APPROVED** [review]
:+1:

**Pull request merged**